tests/data/MerchFactory.ts

```diff
@@ -51,9 +51,25 @@ export class MerchFactory {
         .createOptions(numOptions)
         .map((option) => MerchandiseItemOptionModel.merge(option, { item: fake }));
     }
+    if (!substitute?.photos) {
+      const numPhotos = FactoryUtils.getRandomNumber(1, 5);
+      fake.photos = MerchFactory
+        .createPhotos(numPhotos)
+        .map((photo, i) => MerchandiseItemPhotoModel.merge(photo, { position: i, merchItem: fake }));
+    }
```

Comment:
nit: createPhotos should have the responsibiilty of managing positions, since they're both at the same layer of abstraction (and managing positions is at a lower level of abstraction than creating a merch item). Let's move the positioning logic in there instead

Reply:
Changed. What do you think about moving merchItem to createPhotos as an optional field?